### PR TITLE
Low prio: cleanup code and add tests

### PR DESCRIPTION
**Pull request by alshakero**
Whenever someone faces an issue with `palindrom-redirect` I panic a little because there are no tests and nothing to look at. 

This PR cleans the code a little bit and adds tests.

### CODE REVIEW

File: test/spec-history.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <script src="../../web-component-tester/browser.js"></script>
+
+    <link rel="import" href="../../polymer/polymer.html">
+    <link rel="import" href="../palindrom-redirect.html">
+</head>
+
+<body>
+    <test-fixture id="element">
+        <template>
+            <dom-bind>
```
Comment (tomalec) [inline test/spec-history.html]: Is this `dom-bind` needed here for some reason? or are you just trying to recreate Starcounter's case? If the latter, I'd create two cases:
1. No `dom-bind` setting URL via attribute.
2. `dom-bind` and setting via `domBind.set('prop','/new/index.html') and `url="{{prop}}"`

Or just remove `dom-bind` and test as you do via prop and attr directly.

File: test/spec-redirect.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <script src="../../web-component-tester/browser.js"></script>
+
+    <link rel="import" href="../../polymer/polymer.html">
+    <link rel="import" href="../palindrom-redirect.html">
+</head>
+
+<body>
+    <test-fixture id="element">
+        <template>
+            <dom-bind>
```
Comment (tomalec) [inline test/spec-redirect.html]: as in previos spec

**Review by tomalec — COMMENTED** [review]
Thanks for initiative! Just a small comment on `dom-bind`

File: test/spec-history.html (hunk 1 above)
Comment (alshakero) [reply test/spec-history.html]: There is no need. It is a mistake. In my head I wanted to use it to change the value then used the id and forgot about it 😄

File: test/spec-redirect.html (hunk 2 above)
Comment (alshakero) [reply test/spec-redirect.html]: There is no need. It is a mistake. In my head I wanted to use it to change the value then used the id and forgot about it 😄

**Review by tomalec — APPROVED** [review]
LGTM

**Comment by tomalec** [comment]
Should we add `package-lock.json`, given we remove it from other repos?

**Comment by alshakero** [comment]
I think we should keep it in the package repo itself. But not in repos that depend on it. We need the audits in the repo itself. The problem was the chain affect of the warning showing in the dependant repos for every vulnarable import due to imported `package-lock.json`.

**Pull request merged**